### PR TITLE
paginate tzkt

### PR DESCRIPTION
**Pull request by bennycio**
Changes:

- **Changed** tezos multichain provider functions to paginate tzkt requests

### CODE REVIEW

**Review by Robinnnnn — APPROVED** [review]
🙏

File: service/multichain/tezos/tezos.go [hunk 1]
```diff
@@ -40,6 +40,10 @@ const (
 
 const tezosNoncePrepend = "Tezos Signed Message: "
 
+type tzMetadataAttribute struct {
```
Comment (jarrel-b) [inline service/multichain/tezos/tezos.go:43]: do we still need this type?

File: service/multichain/tezos/tezos.go [hunk 2]
```diff
@@ -171,79 +168,120 @@ func (d *Provider) GetTokensByWalletAddress(ctx context.Context, addr persist.Ad
...
-	defer resp.Body.Close()
-	if resp.StatusCode != http.StatusOK {
-		return nil, nil, util.GetErrFromResp(resp)
-	}
-	var tzktBalances []tzktBalanceToken
-	if err := json.NewDecoder(resp.Body).Decode(&tzktBalances); err != nil {
-		return nil, nil, err
+	offset := 0
+	resultTokens := []tzktBalanceToken{}
+	limit := 1000
```
Comment (jarrel-b) [inline service/multichain/tezos/tezos.go:173]: nitpicky: mind changing this to a constant so its easy to update?

**Review by jarrel-b — APPROVED** [review]

File: service/multichain/tezos/tezos.go (hunk 2 above)
Comment (bennycio) [reply service/multichain/tezos/tezos.go:173]: fo sho!

**Pull request merged**